org.upnproutercontrol.UPnPRouterControl.json

```diff
@@ -0,0 +1,75 @@
+{
+   "app-id":"org.upnproutercontrol.UPnPRouterControl",
+   "branch":"unstable",
```

Comment:
this won't operate on flathub, everything in the default branch gets overriden to be "stable". If you want to test unstable versions then the beta branch is available. I'd remove this for flathub as it can be confusing.

Reply:
Fixed